Classes/FrontEnd/DefaultController.php

```diff
@@ -405,13 +416,17 @@ protected function getListViewHooks(): array
      * @throws \UnexpectedValueException
      *          if there are registered hook classes that do not implement the
      *          \Tx_Seminars_Interface_Hook_EventSingleView interface
+     *
+     * @deprecated will be removed in seminars 4; use ->getSingleViewHookProvider() instead
+
      */
     protected function getSingleViewHooks(): array
     {
         if (!$this->singleViewHooksHaveBeenRetrieved) {
             $hookClasses = $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['seminars']['singleView'];
             if (is_array($hookClasses)) {
                 foreach ($hookClasses as $hookClass) {
+                    GeneralUtility::deprecationLog($hookClass . ' - since seminars 3.0, interface \\Tx_Seminars_Interface_Hook_EventSingleView will be removed in seminars 4.0');
```

Comment:
This line is longer than 120 characters (the hard limit of PSR-12, although I have not enabled that sniff yet). Could you please split it up?